src/EnergyPlus/Boilers.cc

```diff
@@ -233,75 +201,76 @@ namespace Boilers {
             // ErrorsFound will be set to True if problem was found, left untouched otherwise
             GlobalNames::VerifyUniqueBoilerName(
                 DataIPShortCuts::cCurrentModuleObject, DataIPShortCuts::cAlphaArgs(1), ErrorsFound, DataIPShortCuts::cCurrentModuleObject + " Name");
-            Boiler(BoilerNum).Name = DataIPShortCuts::cAlphaArgs(1);
-            Boiler(BoilerNum).TypeNum = DataPlant::TypeOf_Boiler_Simple;
+            auto &thisBoiler = boilers.Boiler(BoilerNum);
```

Comment:
Adding the state class reference to the front of everything was starting to get cumbersome to look at. ```auto &``` added where possible.